src/js/request.js

```diff
@@ -19,13 +41,23 @@ function sendHostname(applicationId) {
     var url = 'https://www.google-analytics.com/collect';
     var hostname = location.hostname;
     var hitType = 'event';
-    var trackingId = 'UA-115377265-9';
+    var eventCategory = 'use';
+    var trackingId = trackingIdMap[applicationId] || trackingIdMap.component;
+    var applicationKeyForStorage = 'TOAST UI ' + applicationId + ' for ' + hostname + ': Statistics';
+    var date = window.localStorage.getItem(applicationKeyForStorage);
 
-    // skip only if the flag is defined and is set to false explicitly
+    // skip if the flag is defined and is set to false explicitly
     if (!type.isUndefined(window.tui) && window.tui.usageStatistics === false) {
```

Comment:
Can be the first condition true? This namespace `tui ` always exist after loading `tui-code-snippet.js`, doesn't it?